src/cmocka/rnp_tests_exportkey.c

```diff
@@ -50,7 +50,7 @@ rnpkeys_exportkey_verifyUserId(void **state)
     rnp_assert_ok(rstate, rnp_key_store_load_keys(&rnp, 1));
```

Comment:
It was quick solution, agree that it is not a good place for utils.h. Also I think we should move some stuff to utils.c as well, but not in this PR. Moved it to the src/lib/utils.h, seems to be a good place.